docker/smqtk_iqr_playground/default_confs/cpu/compute_many_descriptors.json

```diff
@@ -1,6 +1,6 @@
 {
     "descriptor_factory": {
-        "smqtk.representation.descriptor_element.postgres.PostgresDescriptorElement": {
+        "smqtk_descriptors.impls.descriptor_element.postgres.PostgresDescriptorElement": {
```

Comment:
It looks like the GPU version of the config did not get the same updates as this. Same goes for the other GPU-specific config.